astropy/cosmology/flrw/tests/test_w0wzcdm.py

```diff
@@ -156,6 +153,61 @@ def test_comoving_distance_example(self, cosmo_cls, args, kwargs, expected):
         super().test_comoving_distance_example(cosmo_cls, args, kwargs, expected)
 
 
+class TestFlatw0wzCDM(FlatFLRWMixinTest, Testw0wzCDM):
+    """Test :class:`astropy.cosmology.Flatw0wzCDM`."""
+
+    def setup_class(self):
+        """Setup for testing."""
+        super().setup_class(self)
+        self.cls = Flatw0wzCDM
+
+    def test_repr(self, cosmo_cls, cosmo):
+        """Test method ``.__repr__()``."""
+        super().test_repr(cosmo_cls, cosmo)
+
+        expected = (
+            'Flatw0wzCDM(name="ABCMeta", H0=70.0 km / (Mpc s),'
+            " Om0=0.27, w0=-1.0, wz=0.5, Tcmb0=3.0 K,"
+            " Neff=3.04, m_nu=[0. 0. 0.] eV, Ob0=0.03)"
+        )
+        assert repr(cosmo) == expected
+
+    @pytest.mark.skipif(not HAS_SCIPY, reason="scipy is not installed")
+    @pytest.mark.parametrize(
+        ("args", "kwargs", "expected"),
+        [
+            (  # no relativistic species
+                (75.0, 0.3),
+                COMOVING_DISTANCE_EXAMPLE_KWARGS,
+                [3156.41804372, 4951.19475878, 6064.40591021, 6831.18710042] * u.Mpc,
+            ),
+            (  # massless neutrinos
+                (75.0, 0.25),
+                {
+                    **COMOVING_DISTANCE_EXAMPLE_KWARGS,
+                    **{"Tcmb0": 3.0, "Neff": 3, "m_nu": 0 * u.eV},
+                },
+                [3268.38450997, 5205.96494068, 6419.75447923, 7257.77819438] * u.Mpc,
+            ),
+            (  # massive neutrinos
+                (75.0, 0.25),
+                {
+                    **COMOVING_DISTANCE_EXAMPLE_KWARGS,
+                    **{"Tcmb0": 3.0, "Neff": 4, "m_nu": 5 * u.eV},
+                },
+                [2536.77159626, 3721.76294016, 4432.3526772, 4917.90352107] * u.Mpc,
+            ),
+        ],
+    )
+    def test_comoving_distance_example(self, cosmo_cls, args, kwargs, expected):
+        """Test :meth:`astropy.cosmology.LambdaCDM.comoving_distance`.
+
+        These do not come from external codes -- they are just internal checks to make
+        sure nothing changes if we muck with the distance calculators.
+        """
+        super().test_comoving_distance_example(cosmo_cls, args, kwargs, expected)
```

Comment:
Moving `COMOVING_DISTANCE_EXAMPLE_KWARGS` to the function call can shorten the parameter list quite a lot, also in analogous tests.
```suggestion
        super().test_comoving_distance_example(
            cosmo_cls, args, {**COMOVING_DISTANCE_EXAMPLE_KWARGS, **kwargs}, expected
        )
```

Reply:
✅